pkg/cipher/cipher.go

```diff
@@ -0,0 +1,266 @@
+// Package cipher implements common golang encoding interfaces for
+// github.com/skycoin/skycoin/src/cipher
+package cipher
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/skycoin/skycoin/src/cipher"
+)
+
+func init() {
+	cipher.DebugLevel2 = false // DebugLevel2 causes ECDH to be really slow
+}
+
+// GenerateKeyPair creates key pair
+func GenerateKeyPair() (PubKey, SecKey) {
+	pk, sk := cipher.GenerateKeyPair()
+	return PubKey(pk), SecKey(sk)
+}
+
+// GenerateDeterministicKeyPair generates deterministic key pair
+func GenerateDeterministicKeyPair(seed []byte) (PubKey, SecKey, error) {
+	pk, sk, err := cipher.GenerateDeterministicKeyPair(seed)
+	return PubKey(pk), SecKey(sk), err
+}
+
+// NewPubKey converts []byte to a PubKey
+func NewPubKey(b []byte) (PubKey, error) {
+	pk, err := cipher.NewPubKey(b)
+	return PubKey(pk), err
+}
```

Comment:
About this, maybe I don't know its functionality at all, but never convert a text PK to `cipher.PublcKey` at all and make `000...0` as result! So I use this:
```
myPK := cipher.PublicKey{}
myPK.UnmarshalText([]byte("pk_as_text"))
```
I think something is missed here.